.gitignore

```diff
@@ -15,8 +15,7 @@ Icon
 ._*
 
 # Files that might appear in the root of a volume
-.DocumentRevisions-V100
-.fseventsd
+.DocumentRevisions-V100 .fseventsd
```

Comment:
What does this mean exactly?

Reply:
unrelated change?

Reply:
sorry, is unrelated. i deleted an empty line but i think misformatted... i'll revert!